go/keymanager/api/api.go

```diff
@@ -142,10 +143,7 @@ type Genesis struct {
 // SanityCheckStatuses examines the statuses table.
 func SanityCheckStatuses(statuses []*Status) error {
 	for _, status := range statuses {
-		// Verify key manager runtime ID.
-		if !status.ID.IsValid() {
-			return fmt.Errorf("keymanager: sanity check failed: key manager runtime ID %s is invalid", status.ID.String())
-		}
+		// TODO: Verify key manager runtime ID.
```

Comment:
I assume this will be checked when the allocation scheme is implemented?

Reply:
Yeah.